float64 threshold
---
bool result
float64 user_input
